node/src/daemon/setup_reporter.rs

```diff
@@ -835,6 +836,18 @@ impl ValueRetriever for MinHops {
     fn value_name(&self) -> &'static str {
         "min-hops"
     }
+
+    fn computed_default(
+        &self,
+        _bootstrapper_config: &BootstrapperConfig,
+        persistent_config: &dyn PersistentConfiguration,
+        _db_password_opt: &Option<String>,
+    ) -> Option<(String, UiSetupResponseValueStatus)> {
+        match persistent_config.min_hops() {
+            Ok(min_hops) => Some((min_hops.to_string(), Configured)),
+            Err(e) => panic!("Database is corrupt: {:?}", e),
```

Comment:
Also, I think we can write this better, more indicative. When the value from the persistent configuration equals the default you should return `UiSetupResponseValueStatus` being `Default` and not `Configured`. Can you please add in this little logic?

Reply:
Done 👍